api/Document.json

```diff
@@ -296,20 +348,30 @@
       "all": {
         "__compat": {
           "support": {
-            "chrome": {
-              "version_added": "64",
-              "notes": [
-                "Starting in Chrome 65, this property is readonly.",
-                "Before Chrome 64, this property was accessed through the <code>HTMLDocument</code> alias."
-              ]
-            },
-            "chrome_android": {
-              "version_added": "64",
-              "notes": [
-                "Starting in Chrome 65, this property is readonly.",
-                "Before Chrome 64, this property was accessed through the <code>HTMLDocument</code> alias."
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "64",
+                "notes": "Starting in Chrome 65, this property is readonly."
```

Comment:
I don't think this is a useful note to have, but I'll remove it in a separate PR.